docs/mp/config/01_introduction.adoc

```diff
@@ -43,29 +81,88 @@ Step-by-step guide about using {spec-name} in your Helidon MP application.
 --
 ====
 
-== Using {spec-name} in Helidon MP applications
 
-=== Configuration
+=== Using MicroProfile Config Sources and META-INF
+
+Default configuration values can be retrieved from system properties, environment variables, files, directories, URLs or configuration files of the form `${CLASSPATH}/META-INF/microprofile-config.properties`. 
```

Comment:
Not sure if "Default" is correct here - the default is only system properties, environment variables and `microprofile-config.properties`.
When not using the Oxford comma, it is very confusing whether the "files of the form" belong to just the configuration files, or the whole list before that.
I think this deserves a visual list to make it clear, such as:
Configuration can be retrieved from the following configuration sources:

- System properties - a mutable source using `System.getProperties()` to obtain its configuration values
- Environment variables - an immutable source using `System.env()` to obtain its configuration values, also resolves aliases as defined by the MicroProfile Config specification
- `${CLASSPATH}/META-INF/microprofile-config.properties`- properties configuration source as defined by the MP Config spec
- `${CLASSPATH)/application.yaml` - Helidon default configuration source
- File - you can use any properties file on the file system (see `MpConfigSources.create(Path)`)
- URL - you can use a properties source from an URL (see `MpConfigSources.create(URL)`)
- `Map<String, String>` - you can create a source from a Map (see `MpConfigSources.create(Map)`)
- `Properties` - you can create a source directly from properties (see `MpConfigSources.create(Properties)`)
- File on classpath - you can use any properties file on classpath (see `MpConfigSources.classpath(String)`)
- YAML - you can use YAML config sources, as supported by `YamlMpConfigSource`

Reply:
I am not sure when you are referring to (see MpConfigSources.create(Path)). Please let me know what reference should be provided here.